marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -95,13 +97,29 @@ tests = testGroup "Spec.Marconi.ChainIndex.Indexers.Utxo"
           "ToJSON/FromJSON roundtrip for UtxoRow"
           "propJsonRoundtripUtxoRow"
           propJsonRoundtripUtxoRow
+
+    , testGroup "LastSync query"
+        [ testCase
+             "Empty indexer latest sync point is ChainPointAtGenesis"
+             testLastSyncOnFreshIndexer
+
+        , testPropertyNamed
+             "The latest chain point is correct on a indexer that has processed at least a block"
```

Comment:
I have an issue with the term "is correct" :D What is correct? Maybe a description along the lines of: "Querying the latest chain point should return the slot of the last indexed block"?